packages/nx/src/command-line/release/command-object.ts

```diff
@@ -349,10 +356,8 @@ const publishCommand: CommandModule<NxReleaseArgs, PublishOptions> = {
 const planCommand: CommandModule<NxReleaseArgs, PlanOptions> = {
   command: 'plan [bump]',
   aliases: ['pl'],
-  // TODO: Remove this when docs are added
-  // Create a plan to pick a new version and generate a changelog entry.
-  // Hidden for now until the feature is more stable
-  describe: false,
+  describe:
+    'Create a version plan file to specify the desired semver bump for one or more projects or groups, as well as the relevant changelog entry',
```

Comment:
Minor: descriptions should end with a `.`.

Reply:
I was matching the existing commands in there, looks like we are pretty inconsistent in general. We should add a custom lint rule for this like we did for schema descriptions.